solidity/test/support/helpers.js

```diff
@@ -77,7 +70,27 @@ export const toHex = value => {
   return `0x${toHexWithoutPrefix(value)}`
 }
 
-export const deploy = (filePath, ...args) => deployer.perform(filePath, ...args)
+// Convert amount to string, because truffle contracts don't know how to deal
+// with BN's. Should probably remove this, if the issue gets fixed upstream.
+// https://github.com/trufflesuite/truffle/issues/1540
+export const transferLINK =
```

Comment:
It doesn't look like we need these(transferLink/transferLINKAndCall) in most places, the default functionality will work. Can we just call `.toString()` in the few instances where it is failing, or only replace the ones that we need to?

Reply:
OK.